test/functional/server/server_spec.lua

```diff
@@ -38,7 +38,7 @@ describe('serverstart(), serverstop()', function()
     eq('', eval('v:servername'))
 
     -- v:servername will take the next available server.
-    if eval('has("win32")') then
+    if eval('has("win32")') == 1 then
       nvim('command', "call serverstart('\\\\.\\pipe\\test_server_pipe')")
       eq('\\\\.\\pipe\\test_server_pipe', eval('v:servername'))
```

Comment:
I still think we need the extra escaping in the `eq` though if we use single quotes?
Although maybe we can use the brackets there too?

I'm not too familiar with Lua.